pdns/recursordist/docs/settings.rst

```diff
@@ -58,6 +58,35 @@ Note that specifying an IP address without a netmask uses an implicit netmask of
 Like `allow-from`_, except reading from file.
 Overrides the `allow-from`_ setting. To use this feature, supply one netmask per line, with optional comments preceded by a "#".
 
+.. _setting-allow-notify-from:
+
+``allow-notify-from``
+---------------------
+-  IP addresses or netmasks, separated by commas
+-  Default: 127.0.0.0/8, 10.0.0.0/8, 100.64.0.0/10, 169.254.0.0/16, 192.168.0.0/16, 172.16.0.0/12, ::1/128, fc00::/7, fe80::/10
+
+Netmasks (both IPv4 and IPv6) that are allowed to issue NOTIFY queries to the server.
+The default allows access only from :rfc:`1918` private IP addresses.
```

Comment:
This statement is inconsistent with the default value listed a few lines above. 
("Only rfc1918" would be exactly "10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16", but the stated default value includes a number of other networks as well)

I think it'd be better to have an empty default acl, but if this is kept, I think this mismatch needs to be resolved.

Reply:
It is incorrect, but was copied from the existing docs for `allow-from` (and also appears in other places in PowerDNS docs). However, I'm going to change the default to 'no addresses allowed' as you've suggested.